### DISCUSSION TITLE
Anyone know how to use BREAK keyword in this fork?

### DISCUSSION THREAD

**Discussion by ash0080** [vladmandic/sdnext#1086] [Q&A]
```
1girl, red shirt, BREAK, green panties
```
something like this

**Comment by vladmandic** [comment]
exactly like that?

prompt parser breaks the segment and marks it with negative attention:

> Prompt parse-attention: Full parser [['1girl, red shirt,', 1.0], ['BREAK', -1], ['green panties', 1.0]]

**Reply by ash0080** [reply]
> exactly like that?
> 
> prompt parser breaks the segment and marks it with negative attention:
> 
> > Prompt parse-attention: Full parser [['1girl, red shirt,', 1.0], ['BREAK', -1], ['green panties', 1.0]]

Do I need to choose a specific parser? I tried it with Vald but it seems didn't work, and the token count in the upper right corner did not increase. Normally, it would force token counts to move to the next 75 tokens.

**Comment by Anipsy** [comment]
I'm having a strange BREAK behavior as well after last "jumbo patch' from a day ago. (not 100% sure it didn't begin happening sooner)
Whenever i use token in round brackets immediately after BREAK - it will still count tokens as if no BREAK exists in prompt. Here's an [example](https://i.imgur.com/Px2TjOW.png) 
If the first token after BREAK is in round brackets, all the tokens until next BREAK are added to first 75 tokens instead, using round brackets anywhere else but in the first token after break seems fine though.

**Comment by vladmandic** [comment]
fixed in latest version. also added a short-notation `##` which can be used instead of `BREAK`
